imap_processing/swe/decommutation_swe.py

```diff
@@ -0,0 +1,29 @@
+from space_packet_parser import parser, xtcedef
+
+
+def decom_packet(packet_file: str, xtce_packet_definition: str):
```

Comment:
This seems like a very generic decommutator function (nice!) where you can take in anyone's definition. I think we should have everyone follow this then and push it up a level into `imap_processing/decom.py` or something similar. Then import that within the swe module and give it the explicit filepath for swe

```python
from imap_processing import decom


swe_packet_list = decom.decom_packet_file(file_path, swe_packet_definition)
# Then ultra could also use this code easily
ultra_packet_list = decom.decom_packet_file(file_path, ultra_packet_definition)
```

Reply:
That's a good idea. I will change that.